swarm/scheduler/filter.md

```diff
@@ -42,9 +43,9 @@ $ swarm manage --filter=health --filter=dependency
 
 ## Node filters
 
-When creating a container or building an image, you use a `constraint` or
-`health` filter to select a subset of nodes to consider for scheduling.
-If a node in Swarm cluster has a label with key `containerslots`
+When creating a container or building an image, you use a `constraint`,
+`whitelist` or `health` filter to select a subset of nodes to consider for
+scheduling.  If a node in Swarm cluster has a label with key `containerslots`
```

Comment:
Only a single space between sentences, please.

"If a node in a swarm"... missing 'a', no need to capitalize, no need to say both "swarm" and "cluster".